src/components/user/services/userProfileSrv.js

```diff
@@ -1,76 +1,94 @@
 'use strict';
 
 angular.module('znk.infra.user').service('UserProfileService',
-    function (InfraConfigSrv, StorageSrv) {
+    function ($q, ENV) {
         'ngInject';
-        var profilePath = StorageSrv.variables.appUserSpacePath + '/profile';
+
+        var _this = this;
+        var rootRef = new Firebase(ENV.fbDataEndPoint, ENV.firebaseAppScopeName);
+        var refAuthDB = new Firebase(ENV.fbGlobalEndPoint);
```

Comment:
use storagesrv and not firebase directly, see my next comments for more explanations